src/themes/eu/variables/_description-list.scss

```
@@ -3,9 +3,9 @@
 
 $description-list: (
   term-color: var(--c-d-140),
-  term-font: var(--f-l),
-  term-margin-bottom: var(--s-xs),
-  term-margin-top: var(--s-m),
+  term-font: var(--f-m),
+  term-margin-bottom: var(--s-2xs),
+  term-margin-top: var(--s-l),
   definition-font: var(--f-p-m),
```

```suggestion
  definition-font: var(--f-m),
```
font prolonged is deprecated on EU